draft-ietf-dprive-dnsoquic.md

```diff
@@ -558,8 +558,8 @@ section attempts to specify which and how those considerations apply to DoQ.
 
 Historic implementations of DNS clients are known to open and close TCP
 connections for each DNS query. To avoid excess QUIC connections, each with a
-single query, clients SHOULD reuse a single QUIC connection to the recursive
-resolver.
+single query, clients SHOULD send all queries to the same server over a single
+QUIC connection.
 
```

Comment:
Not quite that. There are two issues: one connection at a time, even if several transactions are pending; and, leave connections open rather than closing them immediately, so they can be reused if new transactions start shortly. I agree that this paragraph does not read very well, but I find the proposed fix a bit too concise.